blockchain/sync_manager/Cargo.toml

```diff
@@ -10,16 +10,17 @@ amt = { package = "ipld_amt", path = "../../ipld/amt" }
 blocks = { package = "forest_blocks", path = "../blocks" }
 db = { path = "../../node/db" }
 encoding = { package = "forest_encoding", path = "../../encoding" }
-libp2p =  { git = "https://github.com/SigP/rust-libp2p", rev = "776d13ef046358964c7d64cda3295a3a3cb24743" }
+libp2p = { git = "https://github.com/SigP/rust-libp2p", rev = "776d13ef046358964c7d64cda3295a3a3cb24743" }
```

Comment:
Shouldn't this be `libp2p = "0.15.0"`?

Reply:
yeah I didn't notice, this was auto formatted. Will swap. This must have come in after the swap to libp2p base